lib/users/data.js

```diff
@@ -31,4 +32,27 @@ const modelMerged = (userModel) => {
   return contentTypes([userConfig]);
 };
 
+/*
+ * Cleaner user data post form submission
+ *
+ * @param {object} body req.body post-form submission
+ *
+ * @returns {object} cleaned version of form data
+ */
+const cleanFormData = (body) => {
```

Comment:
Maybe the problem is this is really poorly named? And that it feels like an extra step?

You could very easily do:

```javascript
db('users').insert({
  email: req.body['email--email'],
  role: req.body['email--role'],
  password: bcrypt.hashSync(req.body['email--password']),
});
```

Without this abstraction that's quite a bit confusing